### DISCUSSION TITLE
Custom shortcuts

### DISCUSSION THREAD

**Discussion by dirkjf** [GoogleChromeLabs/ProjectVisBug#546] [Ideas]
Personally, I would prefer not using alt+shift+d to toggle VisBug, but a different shortcut. I think it would be nice to add an option to set custom shortcuts for this and maybe also for the different tools.

**Comment by argyleink** [comment]
you can [do this](https://obie.ai/blog/how-to-add-and-remove-custom-keyboard-shortcuts-and-hotkeys-for-a-chrome-extension-in-5-seconds-flat/) already!

chrome://extensions/shortcuts

![Screen Shot 2022-01-13 at 1 51 04 PM](https://user-images.githubusercontent.com/1134620/149414681-44ef4e4a-ba4d-44bf-bca3-91c24c8cae45.png)

**Reply by dirkjf** [reply]
That's great, I had no idea this was possible.
